src/tailwind.config.js

```diff
@@ -4,8 +4,12 @@ const TailWindTheme = {
   important: true,
   theme: {
     extend: {},
-    fontFamily: {},
-    colors: {},
+    fontFamily: {
+      inter: ['sans-serif'],
+    },
+    colors: {
+      butt: '#e33939',
```

Comment:
rename. This is horrible. imagine if you are maintaining this code and you see "butt"